File: server/EntityLayer/Catalogo/TipoProductoEntity.py
from datetime import datetime

class TipoProductoEntity:
    TipoProductoId: int
    Nombre :str
    FechaRegistro:  datetime
    CodUsuario : str
    Estado: bool


    def Cargar(_DB: any):
        c = TipoProductoEntity()
        c.TipoProductoId = _DB['TipoProductoId']
        c.Nombre = _DB['Nombre']
        # c.FechaRegistro = _DB['FechaRegistro']
        # c.CodUsuario = _DB['CodUsuario']
        # c.Estado = bool(_DB['Estado'])
        return c

 

File: ProyectoORM/server/src/EntityLayer/TipoProductoEntity.ts
import { AccionEnum } from "../Lib/AccionEnum";

export class TipoProductoEntity {
    TipoProductoId: number;
    Nombre: string;
    FechaRegistro: Date;
    CodUsuario: string;
    EstadoRegistro: boolean;
    Action: AccionEnum;
    // FechaRegistroString: string
    // Cont: number
}
